Sources/WebAuthn/Ceremonies/Shared/AuthenticatorData.swift

```diff
@@ -27,11 +27,10 @@ struct AuthenticatorData: Equatable, Sendable {
 }
 
 extension AuthenticatorData {
-    init(bytes: [UInt8]) throws {
+    init(bytes: [UInt8]) throws(WebAuthnError) {
         let minAuthDataLength = 37
-        guard bytes.count >= minAuthDataLength else {
-            throw WebAuthnError.authDataTooShort
-        }
+        guard bytes.count >= minAuthDataLength
+        else { throw .authDataTooShort }
```

Comment:
Am not a fan of `else` being on a separate line 😆 we should get swift format integrated with an agreed style first

Reply:
And here I was being consistent with all my recent commits adhering to this style 🤣
Joking aside, I think this style is much more readable than the assembly of variants I replaced, as conditions can all be aligned, and the else clearly represents a one-line return or throw. When the else case has multiple statements, I agree that it looks better joined to the condition though. Either way, I've undone these fixes and will re-propose them int he context of a format PR 😄